rootfs/scheduler/__init__.py

```diff
@@ -936,62 +936,6 @@ def delete_rc(self, namespace, name):
 
         return response
 
-    def _healthcheck(self, namespace, container, routable=False, path='/', port=5000,
```

Comment:
this is essentially doing the same job as `jsonschema` is in the serializer, but it fails much later in the process *after* someone has already created the healthcheck. I think the way we validate the healthcheck through the serializer is much better, but I'm open to opinions.